Update 12 + 3.2 (#266)
NilPointer-Software/bdfd-wiki: src/bdscript/categoryID.md

# $categoryID
Returns category ID for given category name.

## Syntax
```
$categoryID[category name]
```

## Parameters

> - `category name`: The name of the category from which to retrieve the category ID.

Reviewer: ```suggestion
- `category name`: The name of the category for which to return the ID.
```
